packages/panel/vite.config.ts

```diff
@@ -2,10 +2,14 @@ import reactRefresh from '@vitejs/plugin-react-refresh';
 import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
-import { validateEnvVariables } from './src/common/env';
+import { validateBuildEnv } from './src/common/validateBuildEnv';
 
-export default () => {
-  validateEnvVariables();
+export default async () => {
+  try {
+    await validateBuildEnv();
+  } catch (ex) {
+    throw new Error(ex);
+  }
```

Comment:
Those lines might look like they do nothing because we catch an error and throw it again. However, wrapping this exception in `Error` makes it possible to see the validation message. Below are two console outputs - the first is without the try-catch and the second with it.
<img width="768" alt="CleanShot 2021-08-19 at 01 59 05@2x" src="https://user-images.githubusercontent.com/22938071/129987075-dcef7a8f-65b3-42b4-b5e7-c1c390a6228d.png">